Find a functionally equivalent code:android.widget.CalendarView.getDateTextAppearance:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the text appearance for the calendar dates.
 *
 * @return The text appearance resource id.
 *
 * @attr ref android.R.styleable#CalendarView_dateTextAppearance
 */

@@ -1,3 +1,3 @@
 {
-    return mDateTextAppearanceResId;
+    return mDelegate.getDateTextAppearance();
 }

